ExcelReaderAPI/Controllers/ITRequestUpload.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.IdentityModel.Tokens.Jwt;
+using System.Text;
+using ExcelReaderAPI.Models;
+using ExcelReaderAPI.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.IdentityModel.Tokens;
+
+namespace ExcelReaderAPI.Controllers
+{
+    [Route("it-request")]
+    [ApiController]
+    public class ITRequestUpload : ControllerBase
+    {
+        private readonly FileUploadService _fileUploadService;
+        private readonly IConfiguration _configuration;
+
+        public ITRequestUpload(FileUploadService fileUploadService, IConfiguration configuration)
+        {
+            _fileUploadService = fileUploadService;
+            _configuration = configuration;
+        }
+
+        [Authorize(Roles = "Admin, User")]
+        [HttpGet]
+        public ActionResult<List<ITRequestWithFile>> GetAllITRequests()
+        {
+
+            var token = HttpContext.Request.Headers["Authorization"].FirstOrDefault()?.Split(" ").Last();
```

Comment:
this is not used anywhere

Reply:
it is used, changes were pushed after this comment